front/src/modules/rollingStock/components/rollingStockEditor/RollingStockEditorCurves.tsx

```diff
@@ -273,11 +377,13 @@ export default function RollingStockEditorCurves({
           />
         </div>
         <div className="rollingstock-body">
-          {!isEmpty(selectedCurve[selectedTractionMode]?.curves) && rollingstockParams && (
+          {!isEmpty(selectedCurves[selectedTractionMode]?.curves) && rollingstockParams && (
```

Comment:
Why do you check that the selectedCurves... is not empty ?

```suggestion
          {!isEmpty(selectedCurves[selectedTractionMode]?.curves) && (
```